### PR TITLE
Add FAQ for AWS Throttling

### PR DESCRIPTION
**Pull request by ndowmon**
Added information on how JupiterOne handles rate limiting, and how JupiterOne can further tune the number of requests that we make.

### CODE REVIEW

File: knowledgeBase/APIs_and-integrations/cloud-infrastructure/faqs-aws.md [hunk 1]
```diff
@@ -169,4 +169,10 @@ To omit specific sub-accounts when auto-configuring J1 AWS integrations from an
...
 
-For example, if an AWS account ID is referenced in an IAM policy or S3 bucket policy and that account ID does not match an account integrated with JupiterOne, a mapped entity is created to represent that AWS account with limited metadata. 
\ No newline at end of file
+For example, if an AWS account ID is referenced in an IAM policy or S3 bucket policy and that account ID does not match an account integrated with JupiterOne, a mapped entity is created to represent that AWS account with limited metadata. 
+
+## How does JupiterOne handle rate limiting in the AWS integration?
+
+AWS APIs do not provide proactive rate limit headers. Instead, the JupiterOne AWS Integration tracks the number of API calls made during each integration execution, and attempts to avoid throttling using a Token Bucket (similar to the AWS implementation for rate limiting, described here: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html).
```
Comment (collierkirkland) [inline knowledgeBase/APIs_and-integrations/cloud-infrastructure/faqs-aws.md:176]: ([similar to the AWS implementation for rate limiting](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html)).

File: knowledgeBase/APIs_and-integrations/cloud-infrastructure/faqs-aws.md [hunk 2]
```diff
@@ -169,4 +169,10 @@ To omit specific sub-accounts when auto-configuring J1 AWS integrations from an
...
 
-For example, if an AWS account ID is referenced in an IAM policy or S3 bucket policy and that account ID does not match an account integrated with JupiterOne, a mapped entity is created to represent that AWS account with limited metadata. 
\ No newline at end of file
+For example, if an AWS account ID is referenced in an IAM policy or S3 bucket policy and that account ID does not match an account integrated with JupiterOne, a mapped entity is created to represent that AWS account with limited metadata. 
+
+## How does JupiterOne handle rate limiting in the AWS integration?
+
+AWS APIs do not provide proactive rate limit headers. Instead, the JupiterOne AWS Integration tracks the number of API calls made during each integration execution, and attempts to avoid throttling using a Token Bucket (similar to the AWS implementation for rate limiting, described here: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html). 
+
+If users are still experiencing throttling on their AWS APIs, JupiterOne can further tune the amount of requests per second that the JupiterOne AWS Integration is allowed to make.
```
Comment (collierkirkland) [inline knowledgeBase/APIs_and-integrations/cloud-infrastructure/faqs-aws.md:178]: Does the user need to reach out to J1 for us to apply further tuning?

**Review by collierkirkland — CHANGES_REQUESTED** [review]

File: knowledgeBase/APIs_and-integrations/cloud-infrastructure/faqs-aws.md (hunk 2 above)
Comment (ndowmon) [reply knowledgeBase/APIs_and-integrations/cloud-infrastructure/faqs-aws.md:178]: @collierkirkland yes they do. Any thoughts on what we should say here?

Comment (collierkirkland) [reply knowledgeBase/APIs_and-integrations/cloud-infrastructure/faqs-aws.md:178]: Depending on who the user needs to reach out to, we could provide a link directing them to the proper channel.

Something along the lines of:

"If AWS API throttling continues to impact the JupiterOne AWS integration, contact  [link to proper channel here] for assistance in further tuning the integration to increase the amount of requests per second."

Thoughts?

**Review by collierkirkland — APPROVED** [review]

**Pull request merged**